### DISCUSSION TITLE
アカウントを削除した場合の回答について

### DISCUSSION THREAD

**Discussion by mariori12** [increments/qiita-discussions#589] [要望・アイデア・不具合]
#230 を発端に改修いただいた、以下のリリースノートに記述の事項について申し上げます。

 > https://qiita.com/release-notes/2023-10-04

> まだユーザーが退会してしまった場合は消えてしまう状態なのですが、こちらも追って対応を予定しております。

上記について、今回、自身が初めて目の当たりにし、
「時間の無駄」を体感しましたので「対応の目安」について
具体的な期間・対策等についてご提示いただけましたら幸いです。

既に存在しない為、どのような質問か内容についての概要を記述します。
(問題があれば削除・修正します)

■対象質問の概要
　┗「SpringBootでのwebサイト構築学習時にThymeleafを利用した.htmlを表示できない」

タイトルについて、エラー文の
「org.thymeleaf.exceptions.TemplateInputException: Error resolving template」を丸貼付けし、
内容は、「初心者だから分かりません」という旨の具体的な内容を全く記述していない質問でした。

丸投げだった為、質問のガイドラインを提示したうえで
私自身が検索した結果の中で役立ちそうな記事を模索し、方針について提示したところ
アカウントごと全て削除されておりました。

結果として、該当アカウントについて
何も遡れない状態になっており非常に残念に感じました。

最後に、改めての結論記述になりますが、
「ユーザーが退会した場合」についてどのような対応をする想定なのか
期間及び方針をご提示いただければと存じます。

何卒、宜しくお願い申し上げます。

**Comment by tanaka-asami415** [comment]
@mariori12 
ご不便をおかけして申し訳ありません🙇

直近の開発優先度などの兼ね合いにより、着手はできておりませんが
どういう形で実装するかを検討している最中です。
そのため、時期のお約束はできないのですが、早めに解決できるようにしたいと思っています。

**Comment by nak435** [comment]
「アカウントを削除した場合」とは違いますが、

回答が付いた質問は削除できないように変更されてから、質問内容をごっそり消してしまう方（質問者さんご自身のコメントも消す方）を何度も見ています。

理由はともかく、質問自体を取り消したい方が居ることは事実ですので、「ここに質問することはどうゆことなのか」をしっかり提示して同意を得るなどが必要ではないかと思います。
また、質問自体を取り消す場合の手続きも示す必要があるかも知れません。

現状、質問においては「編集履歴」がありませんが、あった方がよいと思う時も多々あります。

ご参考まで

**Reply by tanaka-asami415** [reply]
@nak435 
ご意見ありがとうございます！

> 理由はともかく、質問自体を取り消したい方が居ることは事実ですので、「ここに質問することはどうゆことなのか」をしっかり提示して同意を得るなどが必要ではないかと思います。
また、質問自体を取り消す場合の手続きも示す必要があるかも知れません。

こちらの内容に関しても、「アカウントを削除した場合でも回答のついた質問は削除しないようにする」際の変更時に併せて改善できるよう、検討を進めております🙇

また、質問の編集履歴ですが、確認していただくことが可能です。
記事ページと場所が異なるので分かりにくいかもしれないのですが、質問のタイトル右上にある「…」より閲覧していただけます🙇

![image (8)](https://github.com/increments/qiita-discussions/assets/103916185/7298f487-52ff-4b7d-a6a7-11b4bc775d80)

**Reply by nak435** [reply]
> また、質問の編集履歴ですが、確認していただくことが可能です。
記事ページと場所が異なるので分かりにくいかもしれないのですが、質問のタイトル右上にある「…」より閲覧していただけます🙇

知りませんでした。ありがとうございます。
